Assets/Scripts/UI/PickupPup/NavPanel/NavigationPanel.cs

```diff
@@ -42,15 +42,18 @@ public void showAdoptButton(bool show)
 
     public void OnMenuClick()
     {
+        
         mainMenu.Toggle();
-        if(dogProfile)
+        if(dogProfile.isActiveAndEnabled)
```

Comment:
I think you still want to check both areas:
`if(dogProfile &&  dogProfile.isActiveAndEnabled)`
This prevents a null ref if `dogProfile` is set to null, as that if statement will then evaluate to false. Same note for the `if` check on line 51